### ISSUE TITLE
Make it workable without password column

### ISSUE THREAD

**Issue by kremsy** [r-a-stone/Laravel-Auth-Bridge#7]
We use a authentication in Laravel without any password Column in the database, so it is a bit of a Problem to choose a Password column then, that's why we need the direct login from laravel on the phpbb forum, in the first version I made that manually over some dirty globals ;).

So what it is needed to make a configuration possible without any password-column.

I personally liked the possability to manage users and groups yourself like you had it in the first version, it's just a bit more flexible.

**Comment by r-a-stone** [comment]
I will be adding the that additional functionality back in (but with a bit more grace) by the middle of august!

Do the phpbb users have no passwords aswell?

If you could give me some additional info about your needs to remove the password requirement ill see what i can do!

**Comment by kremsy** [comment]
It is like we use a special system of authentication on the website, which don't need any password, and we just want to use the session from the Website in the Forum as well as I described before, so we don't need any authentication on the forum itself, and as we don't have any password, we can't give proper passwords to the Forum, however we can just give for example a random generated password on the user creation (maybe save it) but would be cool if the authentication would work than without any password ;).

All what we basically need is creation of user from Laravel user controller, force users into an group. And as soon the user logged in we need just to call a function on the forum or something that he is logged in there as well, same for logout.

Another thing what would be nice would be to set Profilepictures as well in the forum from the laravel controller, but however first the basic system should work properly :)

Thanks for your great work!

**Comment by r-a-stone** [comment]
Sounds like a useful addition ill add it in the coming weeks!

**Comment by kremsy** [comment]
Thanks sounds cool :)

**Comment by kremsy** [comment]
Any news on the progress?

**Comment by r-a-stone** [comment]
Yes, i am almost done with the rewrite and will have it finished this weekend most likely. This will also have the ability to use phpBB as the provider if desired. Any columns can be used to authenticate.
